content/hi/docs/reference/glossary/secret.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Secret
```

Comment:
This should be in Hindi

Reply:
You can use `aka:` to also provide the original name.